internal/component/azure/servicebus/subscription.go

```diff
@@ -270,6 +405,87 @@ func (s *Subscription) handleAsync(ctx context.Context, msg *azservicebus.Receiv
 	}()
 }
 
+func (s *Subscription) handleMultipleAsync(ctx context.Context, msgs []*azservicebus.ReceivedMessage, handler BulkHandlerFunc) {
```

Comment:
THis too looks very similar to `handleAsync`. Can you implement them jointly? Perhaps just change `handleAsync` to accept a list of messages and use a single method?